test/controllers/TransferCompleteControllerSpec.scala

```diff
@@ -38,23 +38,35 @@ class TransferCompleteControllerSpec extends FrontEndTestHelper {
 
   "TransferCompleteController GET" should {
     "render the success page if the export was triggered successfully" in {
-      val client = new GraphQLConfiguration(app.configuration).getClient[gcr.Data, gcr.Variables]()
-      val controller = instantiateTransferCompleteController(getAuthorisedSecurityComponents)
-
-      val consignmentReferenceResponse: gcr.GetConsignment = getConsignmentReferenceResponse
-      val data: client.GraphqlData = client.GraphqlData(Some(gcr.Data(Some(consignmentReferenceResponse))), List())
-      val dataString: String = data.asJson.printWith(Printer(dropNullValues = false, ""))
-      wiremockServer.stubFor(post(urlEqualTo("/graphql"))
-        .willReturn(okJson(dataString)))
+      val transferCompleteSubmit = callTransferComplete("consignment")
+      contentAsString(transferCompleteSubmit) must include("Transfer complete")
+      contentAsString(transferCompleteSubmit) must include("TEST-TDR-2021-GB")
+    }
+  }
 
-      val consignmentId = UUID.randomUUID()
-      val transferCompleteSubmit = controller.transferComplete(consignmentId)
-        .apply(FakeRequest(GET, s"/consignment/$consignmentId/transfer-complete").withCSRFToken)
+  "TransferCompleteController GET" should {
+    "render the success page if the export was triggered successfully for a judgment user" in {
+      val transferCompleteSubmit = callTransferComplete("judgment")
       contentAsString(transferCompleteSubmit) must include("Transfer complete")
       contentAsString(transferCompleteSubmit) must include("TEST-TDR-2021-GB")
     }
   }
 
+  private def callTransferComplete(path: String): Future[Result] = {
```

Comment:
I'm thinking, maybe this method can be split into 2 methods, the first one (line 56-63) called something like `setConsignmentReferenceResponse` and then the other one, the same name as the current method.

Also, line 77 no longer needs to be its own method, you could just replace `getConsignmentReferenceResponse` on line 59 with `new gcr.GetConsignment("TEST-TDR-2021-GB")`